src/scripts/spot.js

```diff
@@ -311,6 +311,13 @@ setVolume = function (level, message) {
     });
 };
 
+var convertEmoji = function (str) {
+    if (str[0] === ':' && str[str.length - 1] === ':') {
+        str = str.replace('-', ' ');
```

Comment:
huh~ can you give an example? (and is spotify fine with you just leaving the colons on the front/end?